### PR TITLE
fix: bump node-resque version

### PR DESCRIPTION
**Pull request by catto**
I updated the version of node-resque.
`v5.3.2` should include the fix of #68.

Ref: https://github.com/taskrabbit/node-resque/pull/246

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
   "dependencies": {
     "config": "^1.26.1",
     "ioredis": "^3.1.4",
-    "node-resque": "5.3.0",
+    "node-resque": ^"5.3.2",
```
Comment (tk3fftk) [inline package.json]: this should be `"^5.3.2"`

Comment (catto) [reply package.json]: oh my...

Comment (catto) [reply package.json]: fixed 🙇

**Comment by jithin1987** [comment]
Thanks @catto for the fix. 👏

**Pull request merged**